src/components/Communication/Communication.tsx

```diff
@@ -115,7 +115,7 @@ export function Communication({
             hAlign="center"
             styles={{
               width: "100%",
-              height: "100vh",
```

Comment:
How will developers who would prefer this component occupy the full viewport height achieve that effect without this?

Reply:
You can use it by specifying a style in the corresponding component parent.

This is because the Provider and its components cascaded the height style.